modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -24,8 +24,11 @@ Open Source users: please report issues in the https://github.com/tinymce/tinymc
 * xref:schema[Schema]
 * xref:ui-and-ux-elements-and-component[UI and UX elements and components]
 
+== TinyMCE 6 core changes
```

Comment:
```suggestion
== Core changes
```
We don't need `TinyMCE 6` here I believe? If we do we should use the asciidoc attributes instead.

Reply:
```suggestion
== {productname} {productmajorversion} core changes
```